Sem. 04/Pract.04/Task16.cpp

```diff
@@ -0,0 +1,16 @@
+// Task 16.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
+int first;
+int second;
+int getRemainder(int first, int second)
+{
+	std::cin >> first >> second;
```

Comment:
You shouldn't read anything from the console. The numbers are passed to the function as parameters.